OpenEdXMobile/src/main/java/org/edx/mobile/view/RegisterActivity.kt

```diff
@@ -59,6 +59,8 @@ class RegisterActivity : BaseFragmentActivity(), MobileLoginCallback {
     private val mFieldViews: MutableList<IRegistrationFieldView> = mutableListOf()
     private lateinit var socialLoginDelegate: SocialLoginDelegate
     private lateinit var errorNotification: FullScreenErrorNotification
+    private var socialRegistrationType = SocialSourceType.UNKNOWN
+    private var savedRegistrationFormState: Bundle? = null
```

Comment:
Please verify the states when the user rotates the screen simultaneously multiple times, the fields lose their state and data.